File: cakk-api/src/main/java/com/cakk/api/config/ComponentScanConfig.java
package com.cakk.api.config;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;

@Configuration
@ComponentScan(basePackages = {
	"com.cakk.client",
	"com.cakk.domain",
	"com.cakk.external",
	"com.cakk.api"
})
public class ComponentScanConfig {
}

File: cakk-api/src/main/kotlin/com/cakk/api/config/ComponentScanConfig.kt
package com.cakk.api.config

import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration

@Configuration
@ComponentScan(
	basePackages = [
		"com.cakk.infrastructure",
		"com.cakk.external",
		"com.cakk.core",
		"com.cakk.api"
	]
)
class ComponentScanConfig
